snack-shop-bff/src/orders/orders.controller.ts

```diff
@@ -0,0 +1,34 @@
+import { Controller, Get } from '@nestjs/common';
+
+@Controller('orders')
+export class OrdersController {
+  @Get()
+  findAll() {
+    const orders = [
+      {
+        _id: '9a4920f6-3297-41e9-a442-b8d4586b7ebe',
```

Comment:
Great choice opting for uuids over enumerable Mongo ObjectIDs! 🎉

Reply:
Remember this is only a mocked example because we don't yet have the backend or mongodb - we need to remember to use UUIDs when we implement this with the DB and the backend service if this is desired.